Library/Formula/loudmouth.rb

```diff
@@ -14,6 +14,21 @@ class Loudmouth < Formula
     sha1 "0b77821cecfa2ea9e3cd9789c21b2e8857f43be0" => :mountain_lion
   end
 
+  head 'https://github.com/mcabber/loudmouth.git'
+
+  head do
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+
+    # Remove this once the following pull request has been applied to master
+    # https://github.com/mcabber/loudmouth/pull/11
```

Comment:
Also can you change the comment to explain what the patch does rather than when to remove it (as that'll become obvious as it won't apply any more)